ui/anduril/ramp-mode.c

```diff
@@ -169,6 +172,15 @@ uint8_t steady_state(Event event, uint16_t arg) {
             //  the "blink at ramp ceil" clause below, over and over)
             if (smooth_steps_in_progress) return EVENT_HANDLED;
         #endif
+        #ifdef BLINK_AT_RAMP_MIDDLE_DELAY
+        if (arg && !cfg.ramp_style && middle_delay_time) {
+            --middle_delay_time;
+            return MISCHIEF_MANAGED;
```

Comment:
`EVENT_HANDLED`

Reply:
Indeed, this commit is from 2019 when it still had the JKR reference. I'll change it.